readme/MacBuild.md

```diff
@@ -1,34 +1,175 @@
-Open a Terminal, and type in `git`, confirm popup from Appstore with "Install" to install development tools.
 
-## Build Libraries (needs to be done once)
-Please take the OS version dependent problems for each step into account. The errors will vary with different versions of the libraries. If you encounter a yet unknown error, feel free to add it to this set of instructions.
+Before you use *any* of the methods listed above, you need to install developer
+tools.
+
+Run `xcode-select --install` in a terminal to install developer tools.
+
+## Xournal++ Mac Homebrew Build
+
+**We do not officially support builds with Homebrew. They are solely for convenience.**
+
+It is highly recommended to either use an official or nightly release.
+Should you still want to build your own version please refer to the app-build above.
+
+### Install Homebrew
+https://brew.sh/
+
+````sh
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+````
+
+### Install dependencies
+````sh
+brew install cmake ninja pkg-config gtk+3 poppler librsvg adwaita-icon-theme libzip portaudio libsndfile
+````
 
-### 1. Build GTK
-````bash
-./build-gtk3.sh
+### Build Xournal++:
+````sh
+git clone http://github.com/xournalpp/xournalpp.git
+cd xournalpp
+mkdir build
+cd build
+cmake .. -GNinja
+cmake --build .
 ````
-#### Potential Errors
-##### itstool (version 2.0.6 on macOS High Sierra 10.13)
+
+## Bundling Xournal++ into a `.app` with gtk-osx
+
+This uses the official [gtk-osx][gtk-osx] project, which builds GTK using the
+jhbuild tool and a curated set of dependencies that allows GTK to work on macOS.
+
+For the easiest build experience, do not have MacPorts or Homebrew installed.
+* If you have any of them installed, you need to create a new user, and use the
+  new user for the whole process. This is because the jhbuild packages are meant
+  to be self-contained, and environment variables from MacPorts or Homebrew
+  configurations may interfere with the jhbuild package builds.
+* Even if you create a new user, packages built by jhbuild may still "find"
+  MacPorts or Homebrew packages. If this is the case, you can either uninstall
+  the corresponding packages; or (for advanced users) you can clone gtk-osx
+  and manually create a jhbuild _moduleset_ that explicitly ensures that such
+  packages are disabled (see the FAQ).
+* One possible way to use jhbuild alongside brew is to unlink all brew modules
+  before running jhbuild. After finishing the build, they can be relinked.
+  Note: this is untested though, and might fail.
+
+### A note on build dependencies
+
+All of the following steps, except for the last one, only need to be done once.
+
+Please take the OS version dependent problems for each step into account.
+The errors will vary with different versions of the libraries.
+If you encounter a yet unknown error, feel free to submit a pull request to add it to this set of instructions.
+
+
+### Step 1: Install jhbuild
```

Comment:
Step 1 should be to clone the `xournalpp` github repository and `cd` into `mac-setup` folder. This step is completely missing here.

Reply:
That doesn't make sense in this context; we can't even build xournalpp without having jhbuild set up first. I have added a sentence clarifying that the reader needs to be logged in as the jhbuild user.

Reply:
I inserted a cloning step between the "build dependencies from gtk-osx" and "build dependencies not in gtk-osx"